qemu/tests/hotplug_virtio_mem.py

```diff
@@ -26,6 +35,10 @@ def run(test, params, env):
     timeout = params.get_numeric("login_timeout", 240)
     threshold = params.get_numeric("threshold", target_type=float)
     error_msg = params.get("error_msg")
+    qemu_path = utils_misc.get_qemu_binary(params)
+    qemu_version = env_process._get_qemu_version(qemu_path)
+    match = re.search(r"[0-9]+\.[0-9]+\.[0-9]+(\-[0-9]+)?", qemu_version)
+    host_qemu = match.group(0)
```

Comment:
I recommend calling a public function to get the qemu version: `virttest.utils_qemu.get_qemu_version` instead of the private function `_get_qemu_version`.

Reply:
@YongxueHong Thanks for the suggestion, code updated !